nixos/tests/all-tests.nix

```diff
@@ -419,7 +419,6 @@ in {
   netdata = handleTest ./netdata.nix {};
   networking.networkd = handleTest ./networking.nix { networkd = true; };
   networking.scripted = handleTest ./networking.nix { networkd = false; };
-  specialisation = handleTest ./specialisation.nix {};
```

Comment:
Does the new test test everything the old test tested?

Reply:
Even more, since the old test had hard-coded paths to specializations (e.g. `/run/current-system/specialisation/inheritconf/bin/switch-to-configuration test`); but in general yeah, we don't lose anything in here.